### PR TITLE
[t9213] S02 70.083 UN Standardized Product Classification on Products

### PR DESCRIPTION
**Pull request by BT-mgomez**
User Stories: 70.083

Tasks: t9213

Notes for reviews: update product-attribute

### CODE REVIEW

File: l10n_eu_product_adr_report/report/product_adr_report.xml [hunk 1]
```diff
@@ -1,91 +1,90 @@
...
-                                <td><b>Vehicle registration:</b></td>
-                                <td style="padding-left: 15px;"></td>
-                            </tr>
-                            <tr>
-                                <td><b>Departure time:</b></td>
-                                <td style="padding-left: 15px;"></td>
-                            </tr>
-                        </table>
+                    <t t-call="web.external_layout">
+                        <img t-if="o.company_id.logo" t-att-src="'data:image/png;base64,%s' % o.company_id.logo" style="max-height:80px;"/>
```
Comment (BT-rmartin) [inline l10n_eu_product_adr_report/report/product_adr_report.xml]: Where is this logo appearing? As you are already using the external layout, maybe this is not longer needed.

File: l10n_eu_product_adr_report/report/product_adr_report.xml [hunk 2]
```diff
@@ -1,91 +1,90 @@
...
             report_type="qweb-pdf"
             name="l10n_eu_product_adr_report.product_adr_report_document"
-            file="l10n_eu_product_adr_report.product_adr_report"
+            file="l10n_eu_product_adr_report.product_adr_report_document"
             attachment_use="False"
         />
 
         <template id="product_adr_report_document">
-
-            <div class="page">
```
Comment (BT-rmartin) [inline l10n_eu_product_adr_report/report/product_adr_report.xml:16]: You shouldn't remove this div. Try  to put it after  <t t-call="web.external_layout">

File: l10n_eu_product_adr_report/report/product_adr_report.xml (hunk 1 above)
Comment (BT-mgomez) [reply l10n_eu_product_adr_report/report/product_adr_report.xml]: You are right, not needed.

File: l10n_eu_product_adr_report/report/product_adr_report.xml (hunk 2 above)
Comment (BT-mgomez) [reply l10n_eu_product_adr_report/report/product_adr_report.xml:16]: Got it back again ;)

**Pull request merged**